src/lib/constants.ts

```diff
@@ -6,6 +6,8 @@ export const API_BASE_URL =
 		? browser
 			? `http://${location.hostname}:11434/api`
 			: `http://localhost:11434/api`
+		: PUBLIC_API_BASE_URL === '/api'
```

Comment:
this is the default browser behavior and should not be set

whether the user sets `/api` or `/ollama-api` or whatever else, it already should and does work with the previous behavior

Reply:
Also, `.origin` won't always exist - such as in private browsing mode, IIRC

Reply:
Agreed, I'll revert the commit and merge this branch to main! Thanks!

Reply:
If you revert this change, I think it's all good to go. I did test it, apparently at the same time you were making these changes.